letsencrypt/client/tests/crypto_util_test.py

```diff
@@ -63,9 +62,9 @@ class MakeKeyTest(unittest.TestCase):  # pylint: disable=too-few-public-methods
 
     def test_it(self):  # pylint: disable=no-self-use
         from letsencrypt.client.crypto_util import make_key
+        # This individual test was taking over 6 seconds...
+        # I have shortened it... to aid debugging the rest of the project
         M2Crypto.RSA.load_key_string(make_key(1024))
-        M2Crypto.RSA.load_key_string(make_key(2048))
-        M2Crypto.RSA.load_key_string(make_key(4096))
```

Comment:
2048 and 4096 are kind of worthless tests. 
1. Being the most popular means that they will already be tested extensively in trial runs.  It would make more sense to test the edge conditions. (atypical key sizes).  
2. We are testing a single line of code that uses another module's API.  Extensively testing someone else's function doesn't seem worth it.  We should be able to read their documentation and follow it.

Here it is: https://www.dlitz.net/software/pycrypto/api/current/Crypto.PublicKey.RSA-module.html

3. It does not make sense for this line of code to take up 25% of the total testing time.  This discourages developers from actually running the tests because they take too long.